addOns/oast/src/main/java/org/zaproxy/addon/oast/services/boast/BoastService.java

```diff
@@ -40,7 +38,7 @@
 
     private List<BoastServer> registeredServers = new ArrayList<>();
     private final ScheduledExecutorService executorService =
-            Executors.newSingleThreadScheduledExecutor(new BoastThreadFactory("ZAP-OAST-BOAST-"));
+            Executors.newSingleThreadScheduledExecutor(new OastThreadFactory("ZAP-OAST-BOAST-"));
```

Comment:
Tweak the thread name string too?

Reply:
This thread is only going to poll the registered BOAST servers, hence it's still BOAST. The Interactsh thread contains Interactsh in the name: [link](https://github.com/zaproxy/zap-extensions/pull/3120/files#diff-b5dc454da8152981512ff7179b4f6bdf35ebdb69d6289f23471d781b0b07b9d8R81).

Reply:
Okay, good.